examples/bigcommerce/README.md

```diff
@@ -203,6 +212,12 @@ const slug = ctx.params?.slug
 const product = await getProduct(Number.parseInt(slug.toString(), 10))
 ```
 
+### Why is the "dynamic product route" using [ISR](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration) when the root catch all route is not?
```

Comment:
This title is a bit inaccurate. ISR and on-demand revalidation are not separate things. In fact, on-demand revalidation is only possible because of ISR—notice that the docs for on-demand revalidation is a section for the ISR docs page. On-demand revalidation is a way to trigger the "R" in ISR on-demand. So really the difference here is that the product pages don't support on-demand revalidation an rely on ISR and a low revalidation period as opposed to just SSG (i.e., if there was no revalidation period)